metrics.yaml

```diff
@@ -9,11 +9,11 @@ startup_event:
     description: |
       The date and time the extension was started up
     bugs:
-      - https://linktobugs.page
+      - https://github.com/mozilla-extensions/firefox-bridge/issues/
```

Comment:
I doubt this is the right link.

Reply:
From [the docs](https://mozilla.github.io/glean/book/reference/yaml/metrics.html#bugs) "A list of bugs (e.g. Bugzilla or GitHub) that are relevant to this metric. For example, bugs that track its original implementation or later changes to it. Each entry should be the full URL to the bug in an issue tracker."

I initially thought it it was just where all the bugs would be made. Reading the description, I'm not sure what to put here. The issue is that no public bugs were made about any metric. Even the Jira bugs were kinda all over the place. The amount of spec changes and quick additions or removals led to a lot of PRs about telemetry, many of which didn't have ANY bug ticket associated.

I can 
1. track down every single jira ticket and PR that had changes to each metric and list it under them

or

2. Make a public bug for each metric group (like install event, startup event) that can act as the hub for future changes

What would you suggest to put here?

Reply:
I'd put the same link for `bugs` here and for `data_reviews`, since the data review is likely the best link to describe all the metrics.

Reply:
So issue #87 for both?